.profile.khan

```diff
@@ -117,6 +117,7 @@ if [ -z "$DO_NOT_ACTIVATE_VIRTUALENV_KHAN27" ]; then
     # This is desireable if you're touching the python monolith
     if [ -s ~/.virtualenv/khan27/bin/activate ]; then
         . ~/.virtualenv/khan27/bin/activate
+        export CLOUDSDK_PYTHON=~/.virtualenv/khan27/bin/python
```

Comment:
Can you comment why this is needed, and add a TODO to remove it once we update our SDK to one that works with python 3.10?  Eventually -- eventually -- we'll move off python2 entirely and this will stop working.

Reply:
I've added comments to the two `CLOUDSDK_PYTHON` exports as well as the gcloud SDK version definition.